curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/6352fcb156834128001ea945.md

```diff
@@ -31,7 +31,7 @@ Your `sumSquaredDifferences` variable should use the `acc` and `el` parameters i
 assert.match(getVariance.toString(), /sumSquaredDifferences\s*=\s*squaredDifferences\.reduce\(\s*function\s*\(\s*acc\s*,\s*el\s*\)/);
 ```
 
-Your `reduce` callback should return the sum of `acc` and `el`.
+You are missing the starting value of the `reduce`.
```

Comment:
but what happens if you have the starting value, but not the return? this looks like it needs to be split in two and the hints and tests redefined